File: Plugins/Aspose_Slides_Java_for_PHP/src/aspose/slides/WorkingWithSmartArt/AddSmartArt.php
<?php
namespace Aspose\Slides\WorkingWithSmartArt;
use com\aspose\slides\Presentation as Presentation;
use com\aspose\slides\SmartArtLayoutType as SmartArtLayoutType;
use com\aspose\slides\SaveFormat as SaveFormat;

class AddSmartArt{

    public static function run($dataDir=null){

        AddSmartArt::create_smartart_shape($dataDir);
    }

    public static function create_smartart_shape($dataDir=null){

        # Create an instance of Presentation class
        $pres = new Presentation();

        # Get the first slide
        $slide = $pres->getSlides()->get_Item(0);

        # Add Smart Art Shape
        $smartArtLayoutType=new SmartArtLayoutType();
        $smart = $slide->getShapes()->addSmartArt(0, 0, 400, 400, $smartArtLayoutType->BasicBlockList);

        # Write the presentation as a PPTX file
        $saveFormat=new SaveFormat();
        $pres->save($dataDir . "SimpleSmartArt.pptx", $saveFormat->Pptx);

        print "Created smartart shape, please check the output file.".PHP_EOL;
    }
}

File: Plugins/Aspose-Slides-Java-for-Jython/asposeslides/WorkingWithSmartArt/AddSmartArt.py
from asposeslides import Settings
from com.aspose.slides import Presentation
from com.aspose.slides import SaveFormat
from com.aspose.slides import SmartArtLayoutType

class AddSmartArt:

    def __init__(self):
              
        self.create_smartart_shape()
        
    def create_smartart_shape(dataDir):
        
        dataDir = Settings.dataDir + 'WorkingWithSmartArt/AddSmartArt/'    
        
        # Create an instance of Presentation class
        pres = Presentation()

        # Get the first slide
        slide = pres.getSlides().get_Item(0)

        # Add Smart Art Shape
        smartArtLayoutType=SmartArtLayoutType
        smart = slide.getShapes().addSmartArt(0, 0, 400, 400, smartArtLayoutType.BasicBlockList)

        # Write the presentation as a PPTX file
        saveFormat=SaveFormat
        pres.save(dataDir + "SimpleSmartArt.pptx", saveFormat.Pptx)

        print "Created smartart shape, please check the output file."


if __name__ == '__main__':        
    AddSmartArt()